.github/workflows/test.yml

```diff
@@ -92,8 +92,12 @@ jobs:
     - name: Windows does not let git check out files with long names
       if: ${{ matrix.os == 'windows-2019'}}
       run: git config --system core.longpaths true
-    - name: run tox
+    - name: run pytest with --benchmark-skip
```

Comment:
It would be better to have only one `- name:` section with both `if` statements, so that our CI runs won't have an "empty" step. Nevertheless, when I do that CI just does not run anything, it gets stuck with all test cases in "expected" state.